harper-core/src/linting/noun_verb_confusion/mod.rs

```diff
@@ -10,9 +10,16 @@ pub(crate) const NOUN_VERB_PAIRS: &[(&str, &str)] = &[
     ("advice", "advise"),
     ("belief", "believe"),
     ("breath", "breathe"),
+    ("device", "devise"),
     ("effect", "affect"), // "Effect" is also a verb meaning "to bring about". "Affect" is a noun in psychology.
     ("emphasis", "emphasize"), // TODO how to handle "emphasise" as well as "emphasize"?
+    ("envelop", "envelope"),
     ("intent", "intend"),
+    ("pour", "poor"),
```

Comment:
"Poor" is not a noun, it's an adjective which can occasionally be used as a noun in "the poor". As such the heurisitcs which are designed to disambiguate nouns and verbs, but not adjectives.

Also you've switched the columns. As you can see from the previous entries, it should be noun first and then verb. I'm surprised your new unit tests worked. Did you add new unit tests?